README.md

```diff
@@ -1,2 +1,10 @@
 # STM32Examples
 Provides several examples for the Arduino core for STM32 MCUs.
+For the complete description of each example, please refer to the comments at the beginning of each .ino file.
+
+Examples for STM32L475VG-DISCOVERY board :
+  * BTLE_sensors_tof_demo.ino :
+	* get environmental data and send it via  BlueTooth to your smartphone.
+	* use Time of Flight sensor to detect gestures
```

Comment:
I just understood tof means Time of Flight maybe put it here or in the example. Time of Flight (tof) 
Or rename to BTLE_sensors_TimeofFligth_demo.ino